node/packages/parser/src/ethereum/index.ts

```diff
@@ -19,12 +19,15 @@ export interface TransactionParserArgs {
   rpcUrl: string
 }
 
+export type Parser = thor.Parser | uniV2.Parser | zrx.Parser
```

Comment:
We should think about making an interface that we can use and conform to for these parsers, especially if we are going to support custom parser injection. Doesn't need to be included in this PR.

Reply:
Agree. I'll do another PR on this 👌